examples/experimental/v1alpha2/grpc-routing/bar-grpcroute.yaml

```diff
@@ -0,0 +1,23 @@
+#$ Used in:
+#$ - site-src/guides/grpc-routing.md
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
```

Comment:
this is called "grpcroute.yaml" but is an httproute, so kind of confusing

Reply:
Indeed. That was a mistake. Fixed and scrubbed the PR for similar errors.